.config/spellcheck.dic

```diff
@@ -230,3 +230,928 @@ XCM
+Composability
+computalbe
+Computerphile
+concat
+conect
+configuraiton
+configuraitons
+Confing
+Conntected
+Consant
+consenssus
+Consensys
+consta
+consts
+Consts
+containerd
+Contentful
+continoty
+Contribor
+controlorigin
+cooldown
+Cooldown
+cooldowns
+Cooldowns
+cooloff
+Cooloff
+corellate
+coreutils
+correspondes
+Cosm
+cosmin
+cosmwasm
+Cosmwasm
+Couldnt
+covderts
+craete
+creater
+crev
+Crocswap
+CROWDLOANREWARDS
+crowdloans
+Crowdloans
+crreate
+Cryptoassets
+cryptoeconomically
+Cryptoni
+ctypes
+cumulatives
+Curency
+curr
+CURRENCYFACTORY
+curv
+customizability
+cypherpunk
+Darkpools
+datatypes
+datetime
+Decentralised
+DECENTRALISED
+decrementor
+Decrementor
+defaultt
+degen
+denoms
+deopsited
+dependancy
+depositable
+Depositable
+depositn
+Deque
+Deregistration
+descript
+Descrition
+desugared
+DHMS
+dicuss
+distro
+Dlab
+Dockerized
+Doesnt
+dont
+Dostama
+dotgraphs
+dotsama
+Dotsama
+doxxed
+drawio
+DTCC
+duplicative
+dustable
+edgecase
+edgeware
+editting
+Editting
+embbed
+emphasising
+Encodable
+endef
+endrosed
+endwhile
+enigne
+entrancy
+enviroment
+envolved
+epoc
+EQLC
+eququal
+ETHBTC
+Etherium
+evaluatable
+everytime
+excempt
+exectuor
+exhastive
+existant
+expireable
+Exstrinsic
+externatlities
+Extesions
+extirnsic
+Extrinsiscs
+extrsinsic
+failes
+feautes
+feeconfig
+feesless
+Fetador
+Finalised
+findutils
+fintech
+fitlered
+flashloans
+fnft
+Fnft
+FNFT
+followin
+fontconfig
+FRAX
+freetype
+Fridman
+frontrunable
+frop
+FSIZE
+fullfilled
+funcitonal
+functionl
+Fundsto
+Gauba
+GENERALISED
+Genrates
+gnugrep
+gnumake
+graphviz
+Guarranteed
+Haider
+halborm
+halborn
+Halborn
+hallborn
+hander
+hapenning
+Hapenning
+hardocded
+hasher
+Hasher
+hashrate
+Hashrate
+hashset
+haskeller
+Haskeller
+hasura
+headerdata
+hexx
+hljs
+horiz
+Horiz
+Hoskinson
+hotjar
+Hotjar
+HOTJAR
+hrfjw
+Hypersphere
+iavl
+idefinitely
+identifer
+iefix
+ifeq
+imagesdir
+imagesoutdir
+immer
+impls
+incentivise
+incentivized
+Incentivized
+incentivizes
+incentivizing
+incetives
+incetivised
+incoroporate
+inected
+Inegration
+inﬂuencers
+inforced
+informations
+initailized
+initailizes
+inital
+instantiator
+intented
+Intented
+interactie
+interchain
+interecosystem
+Intereste
+interoperating
+Interpeter
+interuption
+intialize
+Intransfer
+invulnerables
+Invulnerables
+Irizar
+journalctl
+judgements
+Judgements
+junod
+kaiserkarel
+Karel
+keccak
+Keword
+ksmusdt
+Kubat
+Kulupu
+kusd
+KUSD
+Labelprops
+Lachen
+Lanlege
+Larriba
+lastmod
+layr
+LAYR
+LAYR’s
+lbpool
+lcgfmg
+lcov
+legos
+lendable
+leverageable
+libp
+libsecp
+libwasmvm
+lifecycles
+Lifecycles
+ligh
+lilke
+limitting
+linecap
+linejoin
+liqudated
+liqudations
+liqudity
+Liqudity
+liquity
+LKSM
+lldb
+localchain
+lockvotes
+lppicapbloother
+lstrip
+macroses
+Maping
+Maxmimum
+Mechnism
+medianize
+MEMLOCK
+menaing
+Mergify
+merkle
+Merkle
+mesage
+messagingxcm
+metadatas
+mili
+milli
+millify
+Millisecs
+MILLISECS
+mimimal
+minial
+Minimalistic
+minterd
+mintues
+minvested
+miterlimit
```

Comment:
minterlimit

Reply:
it uses in *.svg
it is a [css attribute](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-miterlimit)